applications/launchpad_v2/src/styles/gradients.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Gradient palette
+ */
+
+export const gradients = {
+  tari: 'linear-gradient(239.91deg, #9330FF 0%, #593A9B 131%)',
```

Comment:
would be nice to use values from `colors.ts` in here, so that we don't have to change them in two places